### PR TITLE
i18n(es): minor grammatical changes

### PR DESCRIPTION
**Pull request by SantaCRC**
Less grammatical corrections because some sentences do not make sense in Spanish.

### CODE REVIEW

File: i18n/es.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 toggleMenu:
-    other: Toggle Menu
+    other: Esconder Menu
```
Comment (CaiJimmy) [inline i18n/es.yaml:2]: As we are modifying this translation, we can change it to the correct spelling `menú

**Review by CaiJimmy — CHANGES_REQUESTED** [review]
Thanks!

File: i18n/es.yaml [hunk 2]
```diff
@@ -60,7 +60,7 @@ search:
         other: Búsqueda
 
     placeholder:
-        other: Teclea algo...
+        other: Escibe algo...
```
[the remark this answers is not in the document]
Comment (SantaCRC) [reply i18n/es.yaml:63]: You are right

File: i18n/es.yaml (hunk 1 above)
Comment (SantaCRC) [reply i18n/es.yaml]: Changes already made

**Review by CaiJimmy — APPROVED** [review]
Thanks!

**Pull request merged**